crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -312,6 +312,23 @@ def create_proposal
     base
   end
 
+  # Simple check if HA clusters report some problems
+  # If there are no problems, empty hash is returned.
+  # If this fails, information about failed actions for each cluster founder is
+  # returned in a hash.
+  # User has to manually clean pacemaker resources before proceeding with the upgrade.
+  def check_cluster_health
+    cluster_founders = NodeObject.find("pacemaker_founder:true AND pacemaker_config_environment:*")
+    ret = {}
+    return ret if cluster_founders.empty?
+    check_if_nodes_are_available cluster_founders
+    cluster_founders.each do |n|
+      ssh_retval = n.run_ssh_cmd('crm status | grep "^Failed actions:"')
```

Comment:
`grep -q` was there originally, but than we thought it might be good to actually show user the real Failed action for given node (see `:stdout` later)